File: Diplom.Client/src/shared/models/quiz/Answer.ts
class Answer {
  constructor(quizId: number, questionId: number, value: object) {
    this.quizId = quizId;
    this.questionId = questionId;
    this.value = value;
  }
  id?: number;
  quizId?: number;
  questionId?: number;
  value?: any;
}

export default Answer;

File: Diplom.Domain/Quiz/Models/Answer.cs
namespace Diplom.Domain.Quiz.Models
{
	public class Answer
	{
		public int Id { get; set; }

		public int QuizId { get; set; }

		public int QuestionId { get; set; }

		public object Value { get; set; }

		public int UserId { get; set; }
	}
}
